src/azure-cli/requirements.py3.windows.txt

```diff
@@ -105,7 +105,7 @@ jsmin==2.2.2
 knack==0.8.0rc2
 MarkupSafe==1.1.1
 mock==4.0.2
-msrest==0.6.18
+msrest==0.6.21
```

Comment:
no change need in setup.py?

Reply:
yes, msrest version is not specified explicitly in setup.py, so `azure-keyvault-administration==4.0.0b3` there will make sure `msrest>=0.6.21`